apps/financial-aid/backend/src/app/modules/file/file.service.ts

```diff
@@ -63,6 +65,18 @@ export class FileService {
     })
   }
 
+  async getSpouseApplicationFiles(
```

Comment:
We could have this function take in fileType, rename it to getApplicationFilesByType. Then it could be used for more purposes then this single case, if we need it in the future, just a thought.